### ISSUE TITLE
1.5.8 Time adjustment icon doesn't work

### ISSUE THREAD

**Issue by mszulen** [FrSkyRC/ETHOS-Feedback-Community#4040]
1.5.8, X20s
The time adjustment icon doesn't do anything. I can select the each section of time (h, m, s), but when touching the "notepad" icon, nothing happens.

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/6932464/1785c051-7029-4ca9-8eb9-2f710b093f12)

**Comment by Wolepo** [comment]
When I press [Enter], the hour is activated and I can adjust it. X10S Express

**Comment by mszulen** [comment]
> When I press [Enter], the hour is activated and I can adjust it. X10S Express

Sure, but the touch icon doesn't work.

**Comment by Wolepo** [comment]
It works in the Sim. Click on a point in the time and it will be highlighted in colour. It can then be changed using the rotary wheel. No keyboard is displayed.

**Comment by mszulen** [comment]
As I've said - yes, it works with dial as you say. My point was it does not work via the touch icon that is next to the time.

**Comment by strgaltdel** [comment]
tested on x18s /1.5.8 latest
result:
1st touch on icon, and you've focus on field
2nd touch on hour /minute / seconds let change corresponding values in edit mode

so you can adjust by using encoder.
 
would be the behavior i expect

**Comment by mszulen** [comment]
My point was the icon next to it doesn't do anything. Either selector should be added or the icon removed.This is inconsistent to the date selector/change.

**Comment by pld74** [comment]
X20S owner here, works fine....

1st touch - time HH:MM:SS turns yellow but not highlighted
2nd touch - highlights SS (seconds) field
Other touches on HH or MM selects those

Works as expected

**Comment by mszulen** [comment]
Let me rephrase it, because apparently, no one understands the point:

I do not mean that selection of time segments doesn't work - this works and then you can use dial to change.
What I mean is that:

1. "notepad" icon next to the time doesn't do anything (either should be removed or the selector pop-up added)
2. The date field above works differently (touching the icon next to the date opens the selector). This is just inconsistent with the time field - 2 date/time fields and each works differently.

**Comment by bsongis-frsky** [comment]
Implement a TimeDialog like we have for the Date

**Comment by mszulen** [comment]
> Implement a TimeDialog like we have for the Date

Thank you! :)

**Comment by mtschaff** [comment]
I'd like to add that this same behavior (clicking on the icon does not result in the virtual keyboard being displayed) also appears in the Timers section.  When configuring a timer, the virtual keyboard icon is displayed, but no keyboard is displayed when the icon is clicked.

**Comment by mawzthefinn** [comment]
Icon is gone in 26.1
